tests/svg_types_test.py

```diff
@@ -33,10 +33,30 @@
         ("m2,2 c1,-1 2,4 3,3 C4 4 5 5 6 6", "M2,2 C3,1 4,6 5,5 C4,4 5,5 6,6"),
         # Elliptic arc that goes haywire when stroked
         ("M7,5 a3,1 0,0,0 0,-3 a3,3 0 0 1 -4,2", "M7,5 A3 1 0 0 0 7,2 A3 3 0 0 1 3,4"),
+        # clock hand's path in which the last point must be == start point when absolutized
+        (
+            "m63.8 63.98h0.4c2.1 0 3.8-1.7 3.8-3.8v-32.4c0-2.1-1.7-3.8-3.8-3.8h-0.4"
+            "c-2.1 0-3.8 1.7-3.8 3.8v32.4c0 2.1 1.7 3.8 3.8 3.8z",
+            "M63.8,63.98 H64.2 C66.3,63.98 68,62.28 68,60.18 V27.78 "
+            "C68,25.68 66.3,23.98 64.2,23.98 H63.8 C61.7,23.98 60,25.68 60,27.78 "
+            "V60.18 C60,62.28 61.7,63.98 63.8,63.98 Z",
```

Comment:
this test only passes because we do `round_floats(3)`. If you try to remove the rounding, you'll see the last point before the final Z is `63.800000000000004,63.98 Z` (instead of `63.8,63.98 Z` above). You'll also notice that if you try to run the nanoemoji tests with this updated (non-rounding) picosvg, they will fail for the same reason (ie. you'll see a new duplicate point appearing at the end of some contours).
That's precisely what commit https://github.com/googlefonts/picosvg/pull/68/commits/e83f31583b343f8a26264b22424c59a01a3a197f in my other PR https://github.com/googlefonts/picosvg/pull/68 is meant to address.

Reply:
Good catch, ty. I'll need another test in pico then.

Reply:
Amended to snap to subpath start per IM. I'm actually wondering if a generalized version makes sense: upon encountering a new point, if it is *very* similar to a point we've seen before snap it to the prior point.

Reply:
> I'm actually wondering if a generalized version makes sense

I doubt it would be useful. The only reason we are doing for the last point is to avoid the extra implicit closing line. I see no reasons to do it all the time

Reply:
Identical points will compress better, but probably that's not enough unless we measure significant wins there.